<?xml version="1.0"?>
<launch>
    <!-- 加载参数文件 -->
    <rosparam command="load" file="$(find navigation)/param/global_planner.yaml" />
    <rosparam command="load" file="$(find navigation)/param/local_planner.yaml" />
    <rosparam command="load" file="$(find navigation)/param/pointcloud_process.yaml" />
    <rosparam command="load" file="$(find navigation)/param/drone_marker.yaml" />
    <rosparam command="load" file="$(find navigation)/param/motion_controller.yaml" />

    <!-- 启动点云处理节点 -->
    <include file="$(find navigation)/launch/pointcloud_process.launch" />

    <!-- 启动全局路径规划节点 -->
    <node pkg="navigation" type="path_planner" name="path_planner" output="screen" />

    <!-- 启动局部路径规划节点 -->
    <node pkg="navigation" type="local_planner" name="local_planner" output="screen" />


    <!-- 启动无人机机身标记节点 -->
    <node pkg="navigation" type="drone_marker" name="drone_marker" output="screen" />
</launch> 